components/contributors/GithubActivity.tsx

```diff
@@ -359,51 +380,67 @@ export default function GithubActivity({ activityData }: Props) {
     .sort(compareByActivityTime);
 
   return (
-    <div className="flex flex-row-reverse flex-wrap items-start justify-between gap-6 md:flex-nowrap">
-      <div className="top-6 my-4 flex w-full flex-col gap-2 rounded-lg border border-primary-500 p-4 md:sticky md:w-64">
-        <h3>Filter Activity</h3>
-        <select
-          className="my-4 block rounded border border-secondary-600 px-2 py-1 text-sm font-medium text-foreground focus:z-10 focus:outline-none dark:border-secondary-300"
-          disabled={!rangePresets}
-          value={rangeQuery}
-          onChange={(event) => {
-            const current = new URLSearchParams(
-              Array.from(searchParams.entries()),
-            );
-            const value = event.target.value;
-            if (!value) {
-              current.delete("range");
-            } else {
-              current.set("range", event.target.value);
-            }
-            const search = current.toString();
-            const query = search ? `?${search}` : "";
-            router.replace(`${pathname}${query}`, { scroll: false });
-          }}
-        >
-          <option value="last-week">Last week</option>
-          <option value="last-month">Last 30 days</option>
-          {rangePresets?.map((preset) => (
-            <option key={preset} value={preset}>
-              {new Date(preset).toLocaleString("default", {
-                month: "long",
-                year: "numeric",
-              })}
+    <div className="flex flex-col justify-center md:flex-row  md:justify-between ">
+      <div className="mr-10 w-full md:w-2/5">
+        <div className="top-20 my-4 flex flex-col gap-2 rounded p-4 shadow-md dark:bg-primary-900 md:sticky">
```

Comment:
IMO the background with primary color does not go well as we don't seem to use primary shade for background.

<img width="1041" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/1b86b69f-ae0b-49eb-af5d-f209c75928e1">

```suggestion
        <div className="top-20 my-4 flex flex-col gap-2 rounded p-4 shadow-md dark:bg-secondary-800 md:sticky">
```